src/plugins/saved_objects/public/save_modal/saved_object_save_modal.tsx

```diff
@@ -405,7 +404,10 @@ export class SavedObjectSaveModal extends React.Component<Props, SaveModalState>
           />
         </EuiFlexItem>
         {this.props.mustCopyOnSaveMessage && (
-          <EuiFlexItem css={{ marginLeft: `-${euiThemeVars.euiSize}` }} grow={false}>
+          <EuiFlexItem
+            css={({ euiTheme }) => ({ marginLeft: `-${euiTheme.size.base}` })}
+            grow={false}
+          >
             <EuiIconTip type="iInCircle" content={this.props.mustCopyOnSaveMessage} />
```

Comment:
This message was added by https://github.com/elastic/kibana/pull/175062. It looks like this code might no longer be reachable, as there seems to be no longer an "Edit" button when viewing managed content (tested with a managed dashboard). There is only a "Duplicate" button.